schemas/2015-05-21-preview/Microsoft.DevTestLabs.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-08-01/Microsoft.DevTestLab.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.DevTestLabs",
+  "description": "Azure DevTest Labs Resource Types",
+  "definitions": {
+    "labs": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": [
+            "Microsoft.DevTestLabs/labs"
+          ]
+        },
+        "apiVersion": {
+          "enum": [
+            "2015-05-21-preview"
+          ]
+        }
+      }
+    },
```

Comment:
please add "required" part to eliminate ambiguity. usually "name", "type", "apiVersion" are required for most of the resource types.